frappe/__init__.py

```diff
@@ -512,6 +514,18 @@ def innerfn(fn):
 
 	return innerfn
 
+def timing(f):
+
+    @wraps(f)
```

Comment:
spaces alert! use tabs

Also, can we have the "utility" version of this method? because a decorator has to be used by changing the codebase.

If we have something like `frappe.performance(method_name, args)`, it can be directly used in the console. Method performance will mostly be tested in the console so it is better to make it console friendly.

Reply:
> If we have something like `frappe.performance(method_name, args)`

`%prun` can be used for this